app/presenters/tasks/capital_assessment.rb

```diff
@@ -0,0 +1,29 @@
+module Tasks
+  class CapitalAssessment < BaseTask
+    include TypeOfMeansAssessment
+
+    def path
+      edit_steps_capital_property_type_path
+    end
```

Comment:
Only certain application types go through the full capital assessment so depending on the case type, they may need to be routed to the [trust fund page](https://github.com/ministryofjustice/laa-apply-for-criminal-legal-aid/blob/main/app/services/decisions/outgoings_decision_tree.rb#L35)